packages/diff-sequence/diff.js

```diff
@@ -27,10 +27,9 @@ DiffSequence.diffQueryUnorderedChanges = function (oldResults, newResults,
     var oldDoc = oldResults.get(id);
     if (oldDoc) {
       if (observer.changed && !EJSON.equals(oldDoc, newDoc)) {
-        var projectedNew = projectionFn(newDoc);
-        var projectedOld = projectionFn(oldDoc);
-        var changedFields =
-              DiffSequence.makeChangedFields(projectedNew, projectedOld);
+        var projectedNew = projectionFn(newDoc, false);
```

Comment:
Agreed, I was mostly just testing to see if the general optimization was sane haha. A better approach here would be to curry in `clone=true` [here](https://github.com/veered/meteor/blob/2f232d80c96e7dfb557ac754803ae054028643bd/packages/minimongo/minimongo.js#L1004) and [here](https://github.com/veered/meteor/blob/2f232d80c96e7dfb557ac754803ae054028643bd/packages/minimongo/minimongo.js#L1125). I believe `query.projectionFn` on those lines is guaranteed to have been produced by `LocalCollection._compileProjection`. 

We should be able to get away with not modifying `diff-sequence` at all.